model/adapter.py

```diff
@@ -485,11 +487,16 @@ def apply_flux_bounds(self):
                 lower_bound = float(np.min(scalar_data))
             else:
                 continue
-            model_metabolite = self.model_metabolite(scalar['id'], '_e')
-            if not model_metabolite:
-                self.missing_in_model.append(scalar['id'])
-                logger.info('Model is missing metabolite {}'.format(scalar['id']))
-                return
-            reaction = list(set(model_metabolite.reactions).intersection(self.model.exchanges))[0]
+            # flux measured for a given boundary reaction for a metabolite
+            if 'id' in scalar:
```

Comment:
instead of checking for the "id" field presence I would suggest having a "type" field which would be "reaction" or "compound"